jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -23,33 +23,820 @@ YUI.add('store', function() {
+                    href={doc}>
+                    Learn more about writing charms&nbsp;&rsaquo;
+                  </a></p>
+              </div>
+          </div>
+      </div>);
+    },
+
+    /**
+      The content for the development tools section of the page
+
+      @method _developmentToolsSection
+      @return {Object} The contents of the section
+    */
+    _developmentToolsSection: function() {
+      return (<div className="row row--border-bottom">
+        <div className="inner-wrapper">
+            <h2>Development tools</h2>
+            <ul className="twelve-col no-bullets equal-height">
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=gitlab">
+                        <div
+                          className="one-col no-margin-bottom">
+                            <img src={this._generateIconPath('gitlab')}
+                              alt=""
+                              className="featured-entity__image" />
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              Gitlab
+                            </h3>
+                            <p>by charmers</p>
+                        </div>
+                    </a>
+                </li>
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=jenkins">
+                        <div
+                          className="one-col no-margin-bottom">
+                            <img
+                              src={this._generateIconPath('jenkins')}
+                              alt=""
+                              className="featured-entity__image" />
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              Jenkins
+                            </h3>
+                            <p>by Creative</p>
+                        </div>
+                    </a>
+                </li>
+            </ul>
+        </div>
+      </div>);
+    },
+
+    /**
+      The content for the featured charms and bundles section of the page
+
+      @method _featuredSection
+      @return {Object} The contents of the section
+    */
+    _featuredSection: function() {
+      return (<div className="row equal-height">
+          <div className="box box--kibana align-center four-col">
+              <img src={this._generateLocalImagePath('kabana-promo.png')}
+                alt="" className="box__image" />
+              <div className="align-bottom">
+                  <h2>Kibana</h2>
+                  <a href="/?store=kibana"
```

Comment:
Is there a reason you didn't use the changeState system? This will clear out all of the state for the other components in the GUI, including any flags, etc.